src/main/java/thePackmaster/patches/energyandechopack/TurnStartCheckPatch.java

```diff
@@ -0,0 +1,27 @@
+package thePackmaster.patches.energyandechopack;
+
+import com.evacipated.cardcrawl.modthespire.lib.*;
+import com.megacrit.cardcrawl.actions.GameActionManager;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import javassist.CtBehavior;
+import thePackmaster.packs.EnergyAndEchoPack;
+
+
+@SpirePatch(
```

Comment:
This patch is fine, but FYI it's a bit simpler to make a prefix patch for `clz = AbstractPlayer.class, method = "applyStartOfTurnRelics"`. Avoids the need to define the Locator.

An example of a patch like I'm describing is `DarkLancePatch`.

Reply:
Unless an effect calls this function 🤔 
Exactly what I would have done in my Relic pack if it was good enough for me to pursue....
I could change it but I don't really see a reason why, does it lose so much in optimisation ?

Reply:
This was just a FYI, so not need to change it.